petar-stoyanov/homework7/db.php

```diff
@@ -0,0 +1,108 @@
+<?php
+	if(isset($_POST['from'])&&isset($_POST['to'])&&isset($_POST['message'])) {
```

Comment:
@petar-stoyanov7 
This is not the right place for handling data submitted. You should have separate page (called "send-message.php" for example to handle the data). The database file should be only concert with working with the data(pushing it to the database and pulling it from there)